packages/trade-widget/src/components/TokenSelector/TokenSelector.tsx

```diff
@@ -0,0 +1,107 @@
+import {
+    Button,
+    Column,
+    Columns,
+    Content,
+    Control,
+    Dropdown,
+    DropdownContent,
+    DropdownDivider,
+    DropdownItem,
+    DropdownMenu,
+    DropdownTrigger,
+    Field,
+    Icon,
+    Image,
+    Label,
+    Select,
+} from 'bloomer';
+import * as _ from 'lodash';
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+import { AssetToken } from '../../types';
+
+import TokenSelectorItem from '../TokenSelectorItem';
+
+import { definedTokensToIcon } from '../TokenSelectorItem/TokenSelectorItem';
+
+interface TokenSelectorPropTypes {
+    onChange?: (token: AssetToken) => void;
```

Comment:
Is there somewhere this component is used an `onChange` is not specified? If not, make it required.